### ISSUE TITLE
Range indication? 

### ISSUE THREAD

**Issue by PitStains** [Backupiseasy/ThreatPlates#246]
Is there a way to fade when a target is not in range?

**Comment by Backupiseasy** [comment]
No, currently there is no way to do that. It has be requested quite often, see here (https://github.com/Backupiseasy/ThreatPlates/issues/113) and I actually have a basic patch that would allow me to add this, but I did not finish it as I am unsure regarding the performance impact. Checking for range constantly for every nameplate is CPU intensive.

**Comment by PitStains** [comment]
Thanks, understandable. I was actually hoping to just have the range indication on the **active** target.

**Comment by Backupiseasy** [comment]
That would reduce CPU impact, true. Once I have more time, I still want to look into it and the lib I would like to use  for it (LibSpellRange). It‘s nothing I can add without some thought I think :)
